doc/api.rst

```diff
@@ -18,9 +18,10 @@ Simulation (:py:mod:`hnn_core`):
    Network
    Cell
    CellResponse
+   pick_connection
 
-Published Models (:py:mod:`hnn_core`):
---------------------------------------
+Network Models (:py:mod:`hnn_core`):
```

Comment:
@ntolley I renamed this so there were not too many categories. Let me know if this is okay for you